homeassistant/components/homekit_controller/device_trigger.py

```diff
@@ -241,7 +244,7 @@ def async_fire_triggers(conn: HKDevice, events: dict[tuple[int, int], Any]):
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>